REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.6
 WeakRefStrings 0.3.0
-Nulls 0.0.5
\ No newline at end of file
+Nulls 0.0.5
+NamedTuples 0.4
```

Comment:
I think this is supposed to be 4.0 if I'm reading their releases + METADATA correctly